### PR TITLE
Migreene patch1

### PR DESCRIPTION
**Pull request by mgreenegit**
This change impacts only 1 file - related to offline configuration of package management.  Based on customer feedback over Twitter.  It is a fairly significant overhaul of that file content.

### CODE REVIEW

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md](https://github.com/PowerShell/PowerShell-Docs/blob/migreene-patch1/gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C12%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705121612014435-1205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
fyi updated my note about the TITLE : metadata because I am deeply, deeply confused about OPS.

File: gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md [hunk 1]
```diff
@@ -6,36 +6,41 @@ author:  jpjofre
 ms.prod:  powershell
 keywords:  powershell,cmdlet,gallery
 ms.date:  2016-10-14
-contributor:  manikb
+contributor:  manikb, mgreenegit
 title:  bootstrapping_nuget_proivder_and_exe
```
Comment (zjalexander) [inline gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md]: DEFINITIVE ANSWER:
Change the title. 
image

>Field: Title. Description: Title of the article. Purpose: (SEO) This is the title that gets displayed on a search engine results page. Only the first 65 characters will be displayed, so keep it brief. Use your main SEO keyword for this topic here.

File: gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md [hunk 2]
```diff
@@ -6,36 +6,41 @@ author:  jpjofre
 ms.prod:  powershell
 keywords:  powershell,cmdlet,gallery
 ms.date:  2016-10-14
-contributor:  manikb
+contributor:  manikb, mgreenegit
```
Comment (mgreenegit) [inline gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md]: updatedfixed

File: gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md (hunk 1 above)
Comment (mgreenegit) [inline gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md]: fixed

File: gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md [hunk 3]
```diff
@@ -6,36 +6,41 @@ author:  jpjofre
...
-Added the logic for bootstrap both NuGet provider and NuGet.exe for publish-operations with single prompt message and bootstrap only NuGet provider for non-publish operations.
+NuGet.exe is not included in the latest NuGet provider.
+For publish operations of either a module or script, PowerShellGet requires the binary executable NuGet.exe.
+Only the NuGet provider is required for all other operations, including *find*, *install*, *save*, and *uninstall*.
+PowerShellGet includes logic to handle either a combined bootstrap of the NuGet provider and NuGet.exe, or bootstrap of only the NuGet provider.
+In either case, only a single prompt message should occur.
+If the machine is not connected to the Internet, the user or an administrator must copy a trusted instance of the NuGet provider and/or the NuGet.exe file to the disconnected machine.
 
-## When NuGet provider is not available
+*Note - Starting with version 6, the NuGet provider is included in the installation of PowerShell.*
```
Comment (mgreenegit) [inline gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md]: fixed

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md](https://github.com/PowerShell/PowerShell-Docs/blob/migreene-patch1/gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C13%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705131350385343-1205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md](https://github.com/PowerShell/PowerShell-Docs/blob/migreene-patch1/gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705151919466846-1205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by zjalexander** [comment]
thanks!

**Comment by saldana** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md](https://github.com/PowerShell/PowerShell-Docs/blob/migreene-patch1/gallery/psget/repository/bootstrapping_nuget_proivder_and_exe.md) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2017%5C5%5C15%5Cbab27324-4c59-a348-1473-9c6d587e4020%5CPullRequest%5C201705152303212270-1205%5Cworkflow_report.html).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**